redis/src/main/java/org/hibernate/ogm/datastore/redis/impl/hash/RedisHashEnumType.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.redis.impl.hash;
+
+
+import org.hibernate.HibernateException;
+import org.hibernate.cfg.NotYetImplementedException;
+import org.hibernate.engine.spi.SessionImplementor;
+import org.hibernate.ogm.model.spi.Tuple;
+import org.hibernate.ogm.type.impl.EnumType;
+import org.hibernate.ogm.util.impl.Log;
+import org.hibernate.ogm.util.impl.LoggerFactory;
+import org.hibernate.type.CustomType;
+
+/**
+ * Enum type that represents the ordinal value as String.
```

Comment:
To be sure, this still supports ordinal *and* string-based mapping of enums, right? It's only that in the ordinal case a string such as "3" would be persisted? If so, I find the comment a bit misleading, as it suggests this *only* writes the ordinal value.